dogs.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Adopt a Dog</title>
+  <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+
+    <div class="grid-item-header">
+      <h1><img id="dog-logo" src="images/adoptadog-logo.svg" alt="a brown dog"></h1>
+      <h2>Seattle's Premiere Dog Adoption Agency</h2>
+      <div class="header">
+        <h3><a href="checkout.html">Checkout</a></h3>
+      </div>
+    </div>
+    <div class="global-nav">
+      <a class="header-link" href="index.html">Home /</a>
+      <a class="header-link" href="dogs.html">  &nbsp;&nbsp;Dogs  /</a>
+      <a class="header-link" href="blog.html">  &nbsp;&nbsp;Blog    /</a>
+    </div>
+
+    <div class="dog-blog">
+    <article class="dog">
```

Comment:
Whoops! Watch your indenting here. Should be aligned with the rest of the dog artcles